index.html

```diff
@@ -4,9 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Python Jokes By Team J.L.J</title>
+  <link rel="stylesheet" href="styles/index.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Fira+Code&family=Inconsolata&family=Source+Code+Pro&display=swap" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <h1> Welcome to Python Jokes!</h1>
+    <!-- <h2 class="team"> Team Jenny X. & Laura C. & Jane K.</h2> -->
+  </header>
+    <nav class="main-nav">
+          <li><a href="./pages/gallery.html">Memes</a></li>
+          <li><a href="./pages/facts.html">Fun Facts</a></li>
+    </nav>
+  <section class="joke-blocks">
+    <div class="joke">How does a Python developer greet their friends?<br>
+    "Hello, world!"</div>
+    <div class="joke">Why do Python programmers prefer dark mode? <br>
+    Because light attracts bugs! </div>
+    <div class="joke">What do you call a snake that writes code? <br>
+    A Python developer! </div>    
+  </section>
+    <section class="team">
+    <h2> @ Team Jenny X. & Laura C. & Jane K. 2025</h2>
+  </section>
```

Comment:
Indentation is off for the opening and closing section tags.

```suggestion
  <section class="team">
    <h2> @ Team Jenny X. & Laura C. & Jane K. 2025</h2>
  </section>
```